src/main/main.cpp

```diff
@@ -94,9 +94,19 @@ auto main(int argc, char *argv[]) -> int {
   std::vector<std::string> input_tokens;
   jucc::lexer::Lexer lexer = jucc::lexer::Lexer();
   int token;
+
   while ((token = lexer.GetToken(ifs)) != jucc::lexer::TOK_EOF) {
     input_tokens.emplace_back(jucc::lexer::Lexer::GetTokenType(token));
   }
+  std::vector<std::string> errors = lexer.GetUndeclaredSymbolErrors();
+  errors.insert(errors.end(), lexer.GetDuplicateSymbolErrors().begin(), lexer.GetDuplicateSymbolErrors().end());
+
+  if (!errors.empty()) {
+    for (auto &e : errors) {
```

Comment:
std::cout << "jucc: "; between line 104 and 105

Reply:
done